File: Matury CKE/2022 Maj/Programowanie/Python/zad4_1.py
#numbers=[i.strip() for i in open('../../Dane_2205/przyklad.txt')]
numbers=[i.strip() for i in open('../../Dane_2205/liczby.txt')]
first_latest_the_same=[i for i in numbers if i[0]==i[-1]]
print(f'4.1. {len(first_latest_the_same)} {first_latest_the_same[0]}')

File: Matury CKE/2022 Maj/Programowanie/C++/zad4_1.cpp
#include <iostream>
#include <fstream>
#include <string>

int main()
{
    std::cout<<"4.1. ";

    // std::ifstream dane("C:/Matury-Informatyka/Matury CKE/2022 Maj/Dane_2205/przyklad.txt");
    std::ifstream dane("C:/Matury-Informatyka/Matury CKE/2022 Maj/Dane_2205/liczby.txt");

    std::string num;

    std::string first = "";
    int nCount = 0;

    while(dane >> num){
        if(num[0] == num[num.length()-1]){
            nCount += 1;
            if(first.size() == 0){
                first = num;
            }
        }
    }

    std::cout << nCount << " " << first;

    return 0;
}
